src/main/java/org/spongepowered/api/data/manipulator/immutable/entity/ImmutableDisabledSlotsData.java

```diff
@@ -0,0 +1,33 @@
+package org.spongepowered.api.data.manipulator.immutable.entity;
+
+import java.util.Map;
+
+import org.spongepowered.api.data.manipulator.ImmutableDataManipulator;
+import org.spongepowered.api.data.manipulator.mutable.entity.DisabledSlotsData;
+import org.spongepowered.api.data.value.immutable.ImmutableValue;
+import org.spongepowered.api.entity.living.ArmorStand;
+import org.spongepowered.api.item.inventory.equipment.EquipmentType;
+
+/**
+ * A {@link ImmutableDataManipulator} for the disabled slots in an
+ * {@link ArmorStand}.
+ * 
+ * A slot can have taking disabled, placing disabled or both.
+ */
+public interface ImmutableDisabledSlotsData
+		extends ImmutableDataManipulator<ImmutableDisabledSlotsData, DisabledSlotsData> {
```

Comment:
Can be one line.

Reply:
@gabizou This line is slightly longer than 120 characters.

Reply:
@Lignium Sponge uses 150, not 120.

Reply:
> This line is slightly longer than 120 characters.

Yeah... for the sake of simplicity, that "code style" hasn't been followed for years. Plenty of lines where that's not followed, but for class declarations like this, with a single extension, I'd say it's *fine to extend past the desired length*.